src/PrMerger.js

```diff
@@ -123,9 +171,13 @@ class PrMerger {
 
 // promises to process all PRs once, hiding PrMerger from callers
 function Step() {
-    let mergerer = new PrMerger();
-    return mergerer.execute();
+    try {
+        let mergerer = new PrMerger();
+        return mergerer.execute();
+    } catch (e) {
+        _LastScan = null;
+        throw e;
+    }
```

Comment:
This is awkward. I suspect forgetPr() will become forgetDelayedPr() so that it can manage the delay, and we will eventually end up with something like this:

```suggestion
    const lastScan = _LastScan;
    _LastScan = null;
    const mergerer = new PrMerger(lastScan);
    _LastScan = mergerer.execute();
    return _LastScan.minDelay();
```

... but we can wait for the next iteration to decide.

Reply:
Ok, let's postpone until the next iteration.

Reply:
I think now is the right time to do this, moving minDelay maintenance into PrMerger.

Reply:
Done.